lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -71,7 +72,7 @@
 )
 
 grpc_msg_queue = queue.Queue()
-DEFAULT_CALL_TIMEOUT = 15
+DEFAULT_CALL_TIMEOUT = 6
```

Comment:
can we reduce it to 5 sec to have lower probability of inconsistent timeout between pipelineD and sessionD?